zer0-os/zOS
# feat: update delete message confirmation modal

### What does this do?
- updates delete message modal confirmation

### Why are we making this change?
- as per figma design

### How do I test this?
- click three dot menu on a message you can edit/delete and select delete to make the delete message modal appear. Screen shot of design below.

### Key decisions and Risk Assessment:
  #### Things to consider:
  1. How will this affect security?
  1. How will this affect performance?
  1. Does this change any APIs?

---
